resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -5,11 +5,12 @@ pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
 
 restartfile=<%= EZBake::Config[:restart_file] %>
 start_timeout=<%= EZBake::Config[:start_timeout] %>
-stop_timeout=60
 dir=$(dirname "$restartfile")
 realname="<%= EZBake::Config[:real_name] %>"
 PIDFILE="/var/run/puppetlabs/${realname}/${realname}.pid"
 
+[ -e "${INSTALL_DIR}/ezbake-functions.sh" ] && . "${INSTALL_DIR}/ezbake-functions.sh"
```

Comment:
If the functions script doesn't exist, then this script won't work because it references stuff from in there.  So is there any point in checking for the existence first?

In any case, I presume this line will cause a (desirable) failure because of the `set +e` if the script doesn't exist?